src/librustc_mir/transform/check_consts/validation.rs

```diff
@@ -511,8 +511,7 @@ impl Visitor<'tcx> for Validator<'_, 'mir, 'tcx> {
                         return;
                     }
                     _ => {
-                        self.check_op(ops::FnCallOther);
-                        return;
+                        span_bug!(terminator.source_info.span, "invalid callee of type {:?}", fn_ty,)
```

Comment:
Extra comma on the end, heh.

Reply:
Oh, also, I don't think you need to change the method, try `self.span` here (courtesy of `visit_source_info` above).

Reply:
Hm, it still seems more consistent to use `visit_terminator` since we also use `visit_statement`.

(I am thinking about, in a separate PR, just removing `visit_terminator_kind` unless someone has a good explanation for why it should even exist.)